app/lib/medical-transcription-analysis-stack.ts

```diff
@@ -498,11 +498,11 @@ export class MedicalTranscriptionAnalysisStack extends cdk.Stack {
     
           methods.forEach(method => {
             apiResource.addMethod(method, undefined, {
-              authorizationType: apigateway.AuthorizationType.NONE//apigateway.AuthorizationType.COGNITO,
-              // authorizer: {
-              //   authorizerId: `${authorizer.ref}`
-              //   //`${theauthorizer.ref}`
-              // }
+              authorizationType: apigateway.AuthorizationType.COGNITO,
+              authorizer: {
+                authorizerId: `${authorizer.ref}`
+                //`${theauthorizer.ref}`
```

Comment:
Can we remove the comment?

Reply:
Sure!

Reply:
https://github.com/aws-samples/medical-transcription-analysis/blob/bb69041e36eed9769f2c959a29f1818e3a0983b5/app/lib/medical-transcription-analysis-stack.ts#L500-L501